CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ Unreleased
 ----------
 * Fixes a bug with `EnsureAuthenticatedLinks` causing deep links to not work [#1549](https://github.com/Shopify/shopify_app/pull/1549)
 * Ensure online token is properly used when using `current_shopify_session` [#1566](https://github.com/Shopify/shopify_app/pull/1566)
-* Added Debug Logs [#1545](https://github.com/Shopify/shopify_app/pull/1545)
+* Added debug logs, you can read more about our logging strategy (here)[https://github.com/Shopify/shopify_app/blob/main/docs/logging.md]. [#1545](https://github.com/Shopify/shopify_app/pull/1545)
```

Comment:
I'm wondering if this is the right approach - I would argue that we should document the logger settings that we make available, but not necessarily the strategy behind it (as in, do developers care _why_ as opposed to _how_ here?). No strong opinion here!

BTW this should probably be a relative link (untested):
```suggestion
* Added debug logs, you can read more about our logging strategy (here)[./docs/logging.md]. [#1545](https://github.com/Shopify/shopify_app/pull/1545)
```